benchmark/Makefile.am

```diff
@@ -35,9 +39,10 @@ $(LOCAL_DIR)/config.h: $(LOCAL_DIR)
 
 $(BENCHOUT): $(LOCAL_DIR)/config.h
 	cd $(LOCAL_DIR) && make clean
-	cd $(LOCAL_DIR) && $(TIMER) make
+	cd $(LOCAL_DIR) && $(TIMER) -o baseline.times make
 	cd $(LOCAL_DIR) && make clean
-	cd $(LOCAL_DIR) && $(TIMER) ../$(EXE) make
+	cd $(LOCAL_DIR) && $(TIMER) -o build-recorder.times ../$(EXE) make
+	$(AWK) -f benchmark $(LOCAL_DIR)/build-recorder.times $(LOCAL_DIR)/baseline.times > $(BENCHOUT)
```

Comment:
Isn't the file named `benchmark.awk` ? 

$(BENCHOUT) should be removed on `clobber`

Reply:
@zvr benchmark / benchmark.awk, it's the same to `awk`.

Reply:
well, better be explicit I guess.